codebase/superdarn/src.lib/tk/cnvmodel.1.0/src/cnvmodel.c

```diff
@@ -365,7 +365,7 @@ struct model *interp_coeffs(int ih, float tilt, float mag, float cang, int imod)
   float alow[8],ahgh[8];
   float mlow[6],mhgh[6];
   float mod_angil[8],mod_angih[8];
-  float mod_levi[6];
+  float mod_levi[6]={0.0};
```

Comment:
Respectfully disagree with this change.  The warning does not occur on all systems as shown by the travis-ci log.  Furthermore, examination of the source code shows that it will never actually be used uninitialized.

Reply:
I don't think I understand the objection. The compiler is throwing an unintialized warning. The change initializes the array to make this warning go away. This doesn't break the code or cause any problems.

If the goal is to eliminate warnings and doing so doesn't break anything, what is the problem?

Reply:
The objection is that I don't see any compiler warnings raised in this library, as evidenced by the travis-ci log from the current build of the develop branch (https://travis-ci.org/SuperDARN/rst/builds/633483490?utm_source=github_status&utm_medium=notification):

```
================================================================================
Compiling Library:/home/travis/build/SuperDARN/rst/codebase/superdarn/src.lib/tk/cnvmodel.1.0
================================================================================
make clean
rm -f *.o
make
make.version /home/travis/build/SuperDARN/rst/codebase/superdarn/src.lib/tk/cnvmodel.1.0/src/..
make.hdr /home/travis/build/SuperDARN/rst/codebase/superdarn/src.lib/tk/cnvmodel.1.0/src/../include /home/travis/build/SuperDARN/rst/include/superdarn 
/home/travis/build/SuperDARN/rst/codebase/superdarn/src.lib/tk/cnvmodel.1.0/src/../include/cnvmodel.h
'/home/travis/build/SuperDARN/rst/codebase/superdarn/src.lib/tk/cnvmodel.1.0/src/../include/cnvmodel.h' -> '/home/travis/build/SuperDARN/rst/include/superdarn/cnvmodel.h'
gcc -fPIC -Wall -O3 -ansi -D_GNU_SOURCE -D_LINUX -I/home/travis/build/SuperDARN/rst/include/base -I/home/travis/build/SuperDARN/rst/include/general -I/home/travis/build/SuperDARN/rst/include/superdarn -c cnvmodel.c    
mkdir -p /home/travis/build/SuperDARN/rst/lib
ar -r /home/travis/build/SuperDARN/rst/lib/libcnvmodel.1.0.a cnvmodel.o
ar: creating /home/travis/build/SuperDARN/rst/lib/libcnvmodel.1.0.a
ld -shared -lm -lz  -o /home/travis/build/SuperDARN/rst/lib/libcnvmodel.1.0.so \
              cnvmodel.o -Bstatic -L/home/travis/build/SuperDARN/rst/lib  
cd /home/travis/build/SuperDARN/rst/lib; ln -sf libcnvmodel.1.0.a \
               libcnvmodel."1".a 
cd /home/travis/build/SuperDARN/rst/lib; ln -sf libcnvmodel.1.0.so \
               libcnvmodel."1".so 
```

And neither do any compiler warnings occur on my local machine.  So again, I respectfully disagree with the proposed changes to fix an issue which does not exist.

Reply:
If you run the latest version of GCC, you will see the compiler warnings.

I don't do unnecessary work by making pointless changes :)

Reply:
@asreimer, to be constructive here, can you post the version number of GCC that you're using to see this so that we can compare?

Reply:
Ok, let's revert this change for now. We can revisit this type of warning when we have agreed on a standard for RST - that can wait until after the next release.

Reply:
Similar to 063ed65 this has now been outdated by 38610b1, right @egthomas?  Care to resolve the conversation and/or code review?